glib/Gir.toml

```diff
@@ -379,12 +380,18 @@ status = "generate"
     name = "idle_add_full"
     ignore = true
     [[object.function]]
+    name = "idle_add_once"
+    ignore = true
+    [[object.function]]
```

Comment:
could use a small comment saying we have that manually implemented? same for timeout_add_once. Maybe even mark them as manual so we can profit from the upstream docs?

Reply:
They have handwritten doc comments, also deleting the comment and switching it to manual doesn't seem to produce any doc either for some reason.